synapse/storage/data_stores/main/schema/full_schemas/README.md

```diff
@@ -1,13 +1,23 @@
 # Building full schema dumps
 
-These schemas need to be made from a database that has had all background updates run.
+These schemas need to be made from a database that has had all background
+updates run. The schemas are used at ...
 
 To do so, use `scripts-dev/make_full_schema.sh`. This will produce
-`full.sql.postgres ` and `full.sql.sqlite` files.
+`full.sql.postgres ` and `full.sql.sqlite` files. It generates SQL files for
+creating a brand new Synapse DB with the latest schema, on both SQLite3 and
+Postgres. It does so by having Synapse generate an up-to-date SQLite DB, then
+running synapse_port_db to convert it to Postgres. It then dumps the contents of
+both.
 
 Ensure postgres is installed and your user has the ability to run bash commands
 such as `createdb`.
 
 ```
 ./scripts-dev/make_full_schema.sh -p postgres_username -o output_dir/
 ```
+
+The exported database structure has to be cleared of obsolete tables:
+Delete the CREATE statements for "sqlite_stat1", "schema_version", "applied_schema_deltas", and "applied_module_schemas".
+
+There are two folders `16` and `54` which contain ...
```

Comment:
which contain... ?

I assume you're referring to the directories in https://github.com/matrix-org/synapse/tree/master/synapse/storage/data_stores/main/schema/full_schemas.

54 is the most recent full-schema snapshot. 16 is a snapshot from 2015 or so, for historical reference,